### PR TITLE
Add months to get random month

### PR DESCRIPTION
**Pull request by shamsher31**
Added method to get the random months so you can use to to print date with name of month like January or in short Jan. eg: Thursday 22 January 2016

### CODE REVIEW

File: random_data.go [hunk 1]
```diff
@@ -265,3 +266,8 @@ func IpV4Address() string {
 func Day() string {
 	return randomFrom(jsonData.Days)
 }
+
+// Returns randome month
```
Comment (Pallinder) [inline random_data.go]: Typo

File: random_data_example_test.go [hunk 2]
```diff
@@ -90,4 +91,14 @@ func ExampleRandomdata() {
 	// Print a random day
 	fmt.Println(Day())
 
+	// Print a month
+	fmt.Println(Month())
+
+	// Print randome date
```
Comment (Pallinder) [inline random_data_example_test.go]: 'nother type

File: README.md [hunk 3]
```diff
@@ -120,6 +121,17 @@ func main() {
 
 	// Print a day
 	fmt.Println(randomdata.Day())
+  
+	// Print a month
+	fmt.Println(randomdata.Month())
+  
+	// Print date like Thursday 22 Aug 2016
+	timestamp := time.Now()
```
Comment (Pallinder) [inline README.md]: Perhaps it would make sense to also wrap this into a convenient method, seems to me as this would be a fairly standard use case.

**Comment by Pallinder** [comment]
Awesome. Added some comments.

**Comment by shamsher31** [comment]
Did you get a chance to look at it.

**Comment by Pallinder** [comment]
Great! 👍

**Pull request merged**